doc/Language/nativecall.pod6

```diff
@@ -1246,6 +1246,20 @@ AF_INET6 SOCK_RAW
 2607:f8b0:4006:800::200e
 =end code
 
+=head1 Platform Specific Notes
+
+=head2 MacOS - DYLD_LIBRARY_PATH is ignored
+
+On MacOSX El Capitan and later the I<System Integrity Protection>, short SIP,
+prevents protected processes from passing through several environment
+variables, among them C<DYLD_LIBRARY_PATH>. The C<env> program, often used in
+I<shebang> lines, is one of those programs. This means that whenever C<env> is
+involved in calling a program the C<DYLD_LIBRARY_PATH> variable will be
+cleared. To work around this effect one has to either make sure no protected
+process is involved (this can be difficult) or disable SIP.
```

Comment:
Sorry for being late. I missed your request for review.

I think we need an addition here stating that custom installations of Rakduo into non-standard paths, like under one's home directory, are the most affected ones. But more details in the link below.

Otherwise looks good. Thanks!